docs/docs-source/docs/modules/ROOT/nav.adoc

```diff
@@ -35,7 +35,10 @@ include::partial$include.adoc[]
 *** xref:develop:use-flink-streamlets.adoc[Using Flink Streamlets]
 **** xref:develop:build-flink-streamlets.adoc[Building a Flink Streamlet]
 **** xref:develop:test-flink-streamlet.adoc[Testing a Flink Streamlet]
-*** xref:develop:streamlet-configuration.adoc[Configuring Streamlets]
+*** xref:develop:cloudflow-configuration.adoc[The Configuration Model]
```

Comment:
```suggestion
*** xref:develop:cloudflow-configuration.adoc[Configuring Streamlets]
```

Reply:
On the same level as Developing Streamlets, I think makes sense.

Reply:
It makes sense to me too. Question; if we change to that title then we have to pages with the same title
** xref:develop:cloudflow-configuration.adoc[Configuring Streamlets]
**** xref:develop:streamlet-configuration.adoc[Configuring Streamlets]

Reply:
streamlet-configuration is not the right title for that anyway, it is the specific feature of Configuration Parameters.
Please rename `**** xref:develop:streamlet-configuration.adoc[Configuring Streamlets]` to `**** xref:develop:streamlet-configuration.adoc[Streamlet Config Parameters]`, probably best to also rename the file to streamlet-config-parameters.adoc @franciscolopezsancho .